package/yast2-ntp-client.changes

```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Thu Jun 30 06:51:52 UTC 2016 - jreidinger@suse.com
+
+- Fix regression: crash when reading cron entry failed
+  ( discovered by sap-ha module ) (bsc#977610)
```

Comment:
It is described: crash when reading cron entry failed - in this case SCR agent return nil and code crash